app/views/application_searches/show.html.erb

```diff
@@ -60,6 +63,9 @@
               <td class="govuk-table__cell">
                 <%= app.current_assignment.user_name if app.current_assignment.assigned? %>
               </td>
+              <td class="govuk-table__cell">
+                <%= app.status == 'submitted' ? 'Open' : 'Other' %>
```

Comment:
I think we could also extract this logic to the en.yml, we could have `submitted`, and the other possible status values as keys as well then just pass it the application status and it would return the right value.

See lines 90-93 in the `en.yml` to get an idea of what I'm thinking 👍